tests/system/src/test/java/test/robot/javafx/scene/SceneChangeShouldNotFocusStageTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package test.robot.javafx.scene;
+
+import javafx.animation.Animation;
+import javafx.animation.KeyFrame;
+import javafx.animation.Timeline;
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.Scene;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.Pane;
+import javafx.stage.Stage;
+import javafx.util.Duration;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import test.util.Util;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class SceneChangeShouldNotFocusStageTest {
+    static Stage stage;
+    static CountDownLatch startupLatch = new CountDownLatch(1);
+
+    @Test
+    void windowShouldRemainIconified() {
+        Util.sleep(1000);
+        assertTrue(stage.isIconified(), "Stage should be iconified");
+    }
+
+    @BeforeAll
+    public static void initFX() throws Exception {
+        new Thread(() -> Application.launch(TestApp.class, (String[]) null)).start();
+        waitForLatch(startupLatch, 10, "FX runtime failed to start.");
```

Comment:
With the changes made in #950, we have a new pattern for launch and exit.
These two lines would be now:  `Util.launch(startupLatch, TestApp.class);`
and similarly exit method would be : `Util.shutdown(stage);`
and `waitForLatch` method can be removed.

Reply:
Done.